testing/side-by-side.tsx

```diff
@@ -39,15 +40,18 @@ const SideBySide = ({
 const styles = {
     sideBySide: {
         display: "flex",
+        flexWrap: "wrap",
         flexDirection: "row",
     },
     leftPanel: {
         paddingRight: "30px",
-        flexGrow: 1,
+        flexBasis: "400px",
+        margin: Spacing.medium_16,
```

Comment:
Do we need both padding and margin on this element?

Reply:
It wasn't working as I intended so I've tweaked the styles to be simpler.